platforms/ios/src/TangramMap/platform_ios.mm

```diff
@@ -94,6 +95,15 @@ void initGLExtensions() {
     return data;
 }
 
+std::string iOSPlatform::getAssetPath(const Tangram::Url& _path) const {
+    if (_path.isAbsolute()) {
```

Comment:
Yeah I was not sure about this, can a developer provide any absolute path in an ios app? If its not possible we can get rid of this!